docs/application/vstools/install.md

```diff
@@ -13,16 +13,16 @@ You need the following components on top of Visual Studio to make Visual Studio
 
 ## Prerequisites
 
-To work with Visual Studio Tools for Tizen, your computer must have:
+To work with Visual Studio Tools for Tizen, your computer must have the following:
 
-- At least 1.5 GB of available disk space
-- Visual Studio 2017 to use Tizen 4.0 and 5.0
-- Visual Studio 2019 to use Tizen 4.0 and higher
-- Visual Studio 2022 to use Tizen 6.5 and higher
+- At least 1.5 GB of available disk space.
+- Visual Studio 2017 to use Tizen 4.0 and 5.0.
+- Visual Studio 2019 to use Tizen 4.0 and 6.5
+- Visual Studio 2022 to use Tizen 4.0 and higher
```

Comment:
To maintain consistency with the other bullet points, the sentence should end with a period. 
```suggestion
- Visual Studio 2022 to use Tizen 4.0 and higher.
```